src/lib/Queue.ts

```diff
@@ -17,14 +28,14 @@ export class LandRequestQueue {
     });
   };
 
-  // returns the list of queued, running and awaiting-merge items as these are the actual "queue" per se
+  // returns the list of queued, running, awaiting-merge, and merging items as these are the actual "queue" per se
   // all the status' we display on the frontend
   public getQueue = async (): Promise<LandRequestStatus[]> => {
     const queue = await LandRequestStatus.findAll<LandRequestStatus>({
       where: {
         isLatest: true,
         state: {
-          $in: ['queued', 'running', 'awaiting-merge'],
+          $in: ['queued', 'running', 'awaiting-merge', 'merging'],
```

Comment:
do we need `'will-queue-when-ready',` too?

Reply:
Nope this is just what should appear on the `current-state` screen.